### PR TITLE
Format setup steps for readability

### PR DESCRIPTION
**Pull request by DevLab2425**
@thescientist13 
As I was going through the setup, felt some clarification could be added. Just my take on the steps. Let me know what you think.

### CODE REVIEW

**Comment by thescientist13** [comment]
Can one of the admins verify this patch?

File: README.md [hunk 1]
```diff
@@ -24,22 +24,37 @@ To run, all that is required is to you have the latest LTS version of [Node][] i
...
-1. Run `yarn install`
-1. Run `yarn run setup` 
-1. Replace the `xxx`'s with your Github username and a GitHub
-_"Personal Access Token"_ with the following scopes:
+1. Install Yarn globally
+   ```bash
+   # may need to run as sudo
+   $ npm install -g yarn@0.21.3
+   ```
+1. Install Yarn dependencies
```
Comment (thescientist13) [inline README.md]: I would probably specify it as such
_Install application dependencies_

Just my $.02

**Comment by thescientist13** [comment]
Nice job @DevLab2425 .  Aside from the clarification around dependencies, I dig it!  Thanks!

**Comment by thescientist13** [comment]
ok to test

File: README.md [hunk 2]
```diff
@@ -24,22 +24,37 @@ To run, all that is required is to you have the latest LTS version of [Node][] i
...
+   ```
+1. Log into your GitHub account and create an [Personal Access Token](https://github.com/settings/tokens) with the following scopes:
+   - repo
+   - admin:org
+   - notifications
+
+   For information on creating an access token, please visit the [GitHub article on creating access tokens](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/)
+1. Open `/src/services/credentials.ts` and replace the `xxx`'s with your GitHub username and a Personal Access Token
+
+Note: The `credentials.ts` file is listed within the `.gitignore` to prevent accidental inclusion within the repository files.
```
Comment (DevLab2425) [inline README.md]: @thescientist13,

Added this line about `.gitignore`. Felt it was worth mentioning since some may be hesitant otherwise.

Comment (thescientist13) [inline README.md]: nice!  good call 👍

**Pull request merged**